llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -978,8 +978,9 @@ static void addVPMetadata(Module &M, Instruction &I,
 
   if (I.getMetadata(LLVMContext::MD_prof)) {
     uint64_t Unused;
-    auto ExistingVD =
-        getValueProfDataFromInst(I, IPVK_IndirectCallTarget, ~0U, Unused);
+    // ~0U means get all available value profile data without any count limit
```

Comment:
This seems like overkill, why do we need them all given that we quit below if it is non-empty? At least for now why not just set this to 1? For eventual merging, it could be increased to one of the max values for this call in ICP. If you change this add a TODO

Reply:
Good point, done.